modules/artifact-activation/src/test/java/org/mule/runtime/module/artifact/activation/internal/deployable/LightweightDeployableProjectModelBuilderTestCase.java

```diff
@@ -89,6 +91,26 @@ public void createDeployableProjectModelWithAdditionalDependenciesInAPlugin() th
     assertThat(descriptor.getArtifactId(), is("derby"));
   }
 
+  @Test
+  @Issue("W-15228973")
+  @Description("When a version is provided using pom properties, include the information when building DeployableProjectModel.")
+  public void buildDeployableProjectModelWithPomProperty() throws Exception {
+    installArtifact(getResourceFolder("apps/lightweight/application-using-additional-libraries"),
```

Comment:
what is you set `versionToParse` as a property to a value different than `0`? It should take that instead. Add a test case for that scenario.